src/librbd/internal.cc

```diff
@@ -3374,7 +3378,8 @@ namespace librbd {
   }
 
   int aio_read(ImageCtx *ictx, const vector<pair<uint64_t,uint64_t> >& image_extents,
-	       char *buf, bufferlist *pbl, AioCompletion *c, int op_flags)
+	       char *buf, bufferlist *pbl, AioCompletion *c, int op_flags,
+	       int force_no_cache)
```

Comment:
Well... with my last commit, I still cannot work with rbd_cache enabled. Still trying to figure out what happend...